esp/src/src/Utility.ts

```diff
@@ -91,6 +91,11 @@ export function espTime2SecondsTests() {
     }, this);
 }
 
+export function convertedSize (intsize){
+    var unitConversion = ["Bytes","KB","MB","GB","TB","PB","EB","ZB","YB"], x = Math.floor(Math.log(intsize)/ Math.log(1024));
+    return (intsize/ Math.pow(1024,x)).toFixed(2) + " " + unitConversion[x];
+}
+
```

Comment:
FYI:  Since this is TypeScript we can improve the overall declaration by adding some types:
```typescript
export function convertedSize(intsize: number): string {
    const unitConversion = ["Bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
    const x = Math.floor(Math.log(intsize) / Math.log(1024));
    return (intsize / Math.pow(1024, x)).toFixed(2) + " " + unitConversion[x];
}
```
Type of thing.